x-pack/plugins/maps/server/sample_data/flights_saved_objects.js

```diff
@@ -143,80 +206,59 @@ const layerList = [
           type: 'DYNAMIC',
           options: {
             field: {
-              name: 'avg_of_FlightTimeMin',
               origin: 'source',
+              name: 'doc_count',
             },
             minSize: 1,
-            maxSize: 32,
+            maxSize: 16,
```

Comment:
Can you change the maxSize to 32 and the minSize to 4 so the markers stand out better?

Reply:
@nreese Sure, I can make this change, but the main reason I reduced the size was to deal with overlapping circles at the zoomed out map. Any thoughts on the tradeoff between those two priorities?

Reply:
I think overlapping circles are better then a nearly empty map in the default view.